### DISCUSSION TITLE
XMLSec 1.3.1

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#670] [Announcements]
The [XML Security Library 1.3.1](https://www.aleksey.com/xmlsec/download.html) release includes the following changes:

- Added "--with-libltdl" option for ./configure to allow custom libltdl installations and deprecated "--enable-crypto-dl" option.
- Added support for cclang compiler on non-MacOSX platforms.
- (xmlsec-openssl) Restored support for LibreSSL and bumped minimum required version to 3.5.0.
- (xmlsec-nss) Restored minimum supported NSS version to 3.35.
-  Several other small fixes ([more details](https://github.com/lsh123/xmlsec/commits/master)).

Thanks to all who reported bugs or sent PRs!

Aleksey

<hr /><em>This discussion was created from the release <a href='https://github.com/lsh123/xmlsec/releases/tag/xmlsec_1_3_1'>XMLSec 1.3.1</a>.</em>

**Comment by vmiklos** [comment]
Nice, works fine on the libreoffice side (which is an improvement from 1.3.0 which triggered enough problems that we needed to revert to xmlsec-1.2). Thanks.

**Reply by lsh123** [reply]
There was a saying long time ago -- don't upgrade Windows until SP2 is out :)

**Comment by 1pavlov** [comment]
Hello @lsh123! Could you please push that version to apt? https://launchpad.net/ubuntu/+source/xmlsec1

**Reply by lsh123** [reply]
You need to ask Ubuntu folks, I don’t maintain any ports.
